### ISSUE TITLE
How can I set some class weights for semantic segmentation?

### ISSUE THREAD

**Issue by otakutyrant** [IDEA-Research/MaskDINO#53]
I have fine-tuned a maskdino model in a semantic segmantation dataset that has four classes: other, person, car, and road. However it performs poorly in person and car these two classes while the others otherwise. So I want to enhance the loss penality for person and car these classes. But how? I read some source code about SetCritation but I am not familiar with some loss functions. I appreciate your help!

**Comment by FengLi-ust** [comment]
Maybe you need to find these two classes add increase its loss weight when calculating the loss.
